frontend/src/components/dashboard/DashboardTransactions.jsx

```diff
@@ -1,94 +1,140 @@
 import Link from "next/link";
 
-import React from "react";
+import React, { Component } from "react";
 
 import TransactionIcon from "../../../public/static/images/icon-t-transactions.svg";
 
 import { Row, Col } from "react-bootstrap";
 
 import Content from "../utils/Content";
-import TransactionsList from "../transactions/TransactionsList";
-
-export default ({ transactions }) => (
-  <Content
-    title={<h2>Recent Transactions</h2>}
-    icon={<TransactionIcon style={{ width: "22px" }} />}
-    size="medium"
-    border={false}
-    className="dashboard-transactions"
-  >
-    <Row className="px-0">
-      <Col xs="1" className="pr-0">
-        <div className="dashboard-blocks-hr-parent">
-          <div className="dashboard-blocks-hr" />
-        </div>
-      </Col>
-      <Col xs="11" className="px-0 dashboard-transactions-list">
-        <TransactionsList
-          transactions={transactions}
-          viewMode="compact"
-          reversed
-        />
-        <Row noGutters>
-          <Col xs="1" className="dashboard-transactions-icon-col" />
-          <Col xs="6">
-            <Link href="transactions">
-              <a className="dashboard-footer">View All</a>
-            </Link>
+import ActionsList from "../transactions/ActionsList";
+
+import TransactionsApi from "../../libraries/explorer-wamp/transactions";
+import FlipMove from "react-flip-move";
+
+export default class extends Component {
+  state = {
+    transactions: this.props.transactions
+  };
+
+  componentDidMount() {
+    this.regularFetchInfo();
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+    this.timer = false;
+  }
+
+  fetchInfo = async () => {
+    const transactions = await new TransactionsApi()
+      .getLatestTransactionsInfo()
+      .catch(() => null);
+    this.setState({ transactions });
+  };
+
+  regularFetchInfo = async () => {
+    await this.fetchInfo();
+    if (this.timer !== false) {
+      this.timer = setTimeout(this.regularFetchInfo, 10000);
+    }
+  };
+
+  render() {
+    const { transactions } = this.state;
+    let reserved = true;
+    let actions = transactions.map(transaction => (
+      <ActionsList
+        key={transaction.hash}
+        actions={transaction.actions}
+        transaction={transaction}
+        viewMode="compact"
+        reversed={true}
+      />
+    ));
+
+    if (reserved) actions.reverse();
```

Comment:
Please, run `npm install` / `yarn` in the repository root folder, so it installs `prettier` and register the git hook to auto-format every commit. Next, re-format the code with `prettier --write` or just recreate a commit (`git commit --amend`) and force-push it (`git push origin auto-refresh-yf --force-with-lease`).

Reply:
FIX IT

Reply:
Ah, hmm, prettier does not fix the single-statement `if` unless you explicitly add braces. I prefer to never use this style, as it is error-prone and usually hard to read (I usually "parse" the whole line after `if` keyword as a condition).

Feel free to use the style you find the best working for you. If prettier is fine with this, I am fine with it as well.